beacon-chain/utils/shuffle.go

```diff
@@ -80,62 +33,77 @@ func SplitIndices(l []uint64, n uint64) [][]uint64 {
 	return divided
 }
 
-// PermutedIndex returns `p(index)` in a pseudorandom permutation `p` of `0...list_size - 1` with ``seed`` as entropy.
+// ShuffledIndex returns `p(index)` in a pseudorandom permutation `p` of `0...list_size - 1` with ``seed`` as entropy.
 // We utilize 'swap or not' shuffling in this implementation; we are allocating the memory with the seed that stays
 // constant between iterations instead of reallocating it each iteration as in the spec. This implementation is based
 // on the original implementation from protolambda, https://github.com/protolambda/eth2-shuffle
-//
+func ShuffledIndex(index uint64, indexCount uint64, seed [32]byte) (uint64, error) {
+	return innerShuffledIndex(index, indexCount, seed, true)
```

Comment:
```suggestion
	return innerShuffledIndex(index, indexCount, seed, true /* dir */)
```

I suggest a better name than `dir` not sure what it means in this context

Reply:
added comment for better readability

Reply:
thanks